src/core/login-core.js

```diff
@@ -0,0 +1,219 @@
+import _ from "lodash";
+import { throwStatus } from "../util/express";
+import crypto from "crypto";
+
+import jwt from "jwt-simple";
+const { knex } = require("../util/database").connect();
+require("../init-env-variables");
+
+function tokenForUser(id) {
+  const timestamp = new Date().getTime();
+  return jwt.encode({ sub: id, iat: timestamp }, process.env.JWT_SECRET);
```

Comment:
Should these tokens have expirations time associated with them? Currently, every token issued is valid for eternity, which is a bit problematic